src/main/webapp/index.jsp

```diff
@@ -1,42 +1,65 @@
 <!DOCTYPE html>
+
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
+
 <%@ page import="com.google.sps.HelloAppEngine" %>
+
 <!-- [START_EXCLUDE] -->
+
 <!-- [END_EXCLUDE] -->
+
 <html>
     <head>
+       <script src="chat.js"></script>
```

Comment:
nit: Usually your application's code should be included after all the libraries. Consider moving this after line 25 for authentication.js